### DISCUSSION TITLE
Problema caddy su rasp OS

### DISCUSSION THREAD

**Discussion by qvlogan88** [indomus/forum#3263] [Networking]
salve,

ho installato caddy su docker con l'obiettivo di accedere alla mia istanza ha da remoto
ma dal log vedo che non riesce a risolvere il dns e non capisco quale sia il motivo:

'''
DNS problem: SERVFAIL looking up AAAA for xxxxxxxx.duckdns.org - the domain's nameservers may be malfunctioning","instance":"","subproblems":[]},"order":"[https://acme-staging-v02.api.letsencrypt.org/acme/order/xxxxxxxx/xxxxxxxxxxx","attempt":2,"max_attempts":3}](https://acme-staging-v02.api.letsencrypt.org/acme/order//xxxxxxxx/xxxxxxxxxxx%22,%22attempt%22:2,%22max_attempts%22:3%7D)

{"level":"error","ts":1729879282.706549,"logger":"tls.obtain","msg":"could not get certificate from issuer","identifier":"xxxxxxxxxxx.duckdns.org","issuer":"acme-staging-v02.api.letsencrypt.org-directory","error":"HTTP 400 urn:ietf:params:acme:error:dns 

'''

**Comment by snakuzzo** [comment]
Noi non possiamo dirti qual è il motivo.
Verifica che la risoluzione pubblica del tuo dominio duckdns funzioni correttamente (e non a singhiozzo come negli ultimi tempi).
Inoltre, consiglio spassionato, al posto di usare duckdns e un reverse proxy utilizza cloudflared.

**Reply by qvlogan88** [reply]
grazie per la risposta rapida

il dns va, la risoluzione funziona ma il problema credo sia sul record AAAA che non restituisce nulla, come tra l'altro indica anche il log di caddy

comunque provo con cloudflare piuttosto che impazzire dietro a duckdns
